src/lib.rs

```diff
@@ -17,7 +17,7 @@
 //! Write markdown to TTYs.
 
 #[cfg(feature = "resources")]
-use url;
+extern crate url;
```

Comment:
Why's this change?  Shouldn't `extern crate` be redundant in Rust 2018 edition?

Reply:
Is it perhaps because we now have `url.rs` here?  If so, perhaps we can do things a bit different to avoid this naming conflict :)

Could you try to merge our `url.rs` with the existing `resources.rs`?

Reply:
Yes - the internal module `url.rs` causes that naming conflict. I'll merge it into the existing `resources.rs`.

Reply:
Merged ✅